recipes/reviewer/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.2.7" %}
+{% set sha256 = "7b77536f6c3cda16e08f53eba44d3329bb9bc44356f04644a765b6fc79fdbd31" %}
+
+package:
+  name: reviewer
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/Illumina/REViewer/releases/download/v{{ version }}/REViewer-v{{ version }}-linux_x86_64.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  skip: True # [osx]
+  run_exports:
+    - {{ pin_compatible('reviewer', max_pin="x.x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - zlib
```

Comment:
`zlib` should be in `requirements > host` and `requirements > run`
See https://bioconda.github.io/contributor/guidelines.html#c-c

Reply:
Fixed. Do you think the PR can be merged as it is now? Thanks for assisting!